test/test.kitchen.ts

```diff
@@ -20,13 +20,24 @@ import {ncp} from 'ncp';
 import * as pify from 'pify';
 import * as tmp from 'tmp';
 
-const rename = pify(fs.rename);
```

Comment:
This generally feels like a bug in node.js @ofrobots @mylesborins.  Would you mind filing a bug in their repo and linking to it here?

Reply:
That's not a bug, rather a feature. Node.js documentation clearly states that `fs.rename` is 'Asynchronous `rename(2)`.' `rename(2)` [manual](http://man7.org/linux/man-pages/man2/rename.2.html) states it will return error: 
```
EXDEV  oldpath and newpath are not on the same mounted filesystem.
```

Reply:
Haha bull hockey :) The original intent may have been to bind themselves to unix, but that's an awful limitation on the `fs` API.  This is a usability bug :)

Reply:
I guess this suggests that `fs` is a bit too low level. It is not clear to me that Node.js needs to provide the low level API as well as a high level usable API. Node core *does* have a strong force of standarization – as there isn't any other higher level library that people by default reach out to (e.g. `lodash` but for `fs`). Perhaps we need to do a better job steering users towards higher level alternatives.

Reply:
I understand why it's implemented this way, I just happen to disagree with the decision :)  Good conversation happening over here: 
https://github.com/nodejs/node/issues/19077